specification/trafficmanager/resource-manager/Microsoft.Network/stable/2018-04-01/trafficmanager.json

```diff
@@ -0,0 +1,1578 @@
+{
```

Comment:
Is this spec being referenced in a config file? There must be atleast one tag associated with this to generate the SDKs

Reply:
I don't understand this question. Can you clarify?

Reply:
@allencal is this file was referenced from `readme.md` file? All spec files has to be referenced from `readme.md` file. Otherwise, they will be ignored.

Reply:
Thanks. Added readme.md and readme.ruby.md.